src/units/abilities.cpp

```diff
@@ -2363,19 +2362,36 @@ std::string substitute_variables(const std::string& str, const std::string& tag_
 	return str;
 }
 
+int individual_value(const config::attribute_value *v, int def, const unit_ability & ability, const map_location& loc, const const_attack_ptr& att) {
+	int value = std::round(get_single_ability_value(*v, static_cast<double>(def), ability, loc, att, [&](const wfl::formula& formula, wfl::map_formula_callable& callable) {
+		callable.add("base_value", wfl::variant(def));
+			return std::round(formula.evaluate(callable).as_int());
+	}));
+	return value;
+}
+
+int individual_double_value(const config::attribute_value *v, int def, const unit_ability & ability, const map_location& loc, const const_attack_ptr& att) {
```

Comment:
Please rename to `individual_value_double`.

Reply:
done